dts/bindings/disk/zephyr,ram-disk.yaml

```diff
@@ -31,5 +31,15 @@ properties:
     type: phandle
     description: |
       Optional phandle to the memory region to be used as a RAM disk,
-      if not used a local buffer is defined for each disk instance.
-      Use it with caution as it makes memory contents easily accessible.
+      if not used, and 'disk-image' property is also not used, a local buffer is
+      defined for each disk instance. Use it with caution as it makes memory
+      contents easily accessible.
+
+  disk-image:
```

Comment:
`zephyr,disk-image-path`

this is probably one of the worst offenses of DT mis-usage I've ever seen but, as usual, there is no alternative, so I request you change the property name to this (`zephyr,` prefix is used to track technical debt that is project-specific software configurations, and `-path` is to make it more clear from just the name what it is)

Reply:
@decsny I have made the requested updates.

Reply:
you need to update the original commits to add these changes, not add more with those changes

Reply:
@nordicjm I did make that specific update in the original commit. However subsequent overall updates have been in separate commits. I was a bit unsure of replacing the initial/previous commit in these updates, as contribution guidelines also mention multiple commits in a single PR. Anyway, I have combined (squashed) the commits to a single commit now, that reflects the latest state of the proposed enhancement.